### PR TITLE
master-qa-50759

### PR DESCRIPTION
**Pull request by jaydawu**
https://issues.liferay.com/browse/LRQA-50759
Submit for master.

### CODE REVIEW

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by Sharryxue** [comment]
Just staring review.

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase [hunk 1]
```diff
@@ -1068,13 +1068,79 @@ definition {
...
-	@ignore = "Test Stub"
+	@description = "This test publishes a staged entry to live after deleting live entry, test with blog entries."
 	@priority = "3"
 	test CanPublishToLiveADeletedLiveEntry {
+		task ("Given add staged entry with related asset and publish to live this entry.") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "userea@liferay.com",
+				userLoginFullName = "userfn userln");
+
+			JSONStaging.enableLocalStaging(groupName = "Guest");
```
Comment (Sharryxue) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase:1079]: We'd better use a new site to enable the stage testing.

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase [hunk 2]
```diff
@@ -1068,13 +1068,79 @@ definition {
...
+
+			BlogsNavigator.openBlogsAdmin(siteURLKey = "guest-staging");
 
-		// TODO LRQA-50759 Publish to live a deleted live entry (Message Board)
+			Blogs.addEntry(
```
Comment (Sharryxue) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase:1083]: Use JSONBlog macro to add the entry could speed up the test.

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase [hunk 3]
```diff
@@ -1068,13 +1068,79 @@ definition {
...
+				confirmMessage = "Are you sure you want to publish to live?",
+				entry = "Blogs Entry Title");
+		}
 
+		task ("When delete only the live entry using Personal Data Erasure.") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "test@liferay.com",
+				userLoginFullName = "Test Test");
+
+			Navigator.openStagingSiteURL(defaultSite = "true");
```
Comment (Sharryxue) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase:1097]: This probably not necessary for delete personal data.

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase [hunk 4]
```diff
@@ -1068,13 +1068,79 @@ definition {
...
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "userea@liferay.com",
+				userLoginFullName = "userfn userln");
+
+			BlogsNavigator.openBlogsAdmin(siteURLKey = "guest");
+
+			Blogs.viewNoEntryCP();
+		}
+
+		task ("When publish to live entry again.") {
```
Comment (Sharryxue) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase:1131]: We not using multi When Then When Then while applying the BDD format. Either combining the steps into a When or using a And to nested it.

**Review by Sharryxue — CHANGES_REQUESTED** [review]
Hello @jaydawu 
Thanks for adding the test, and good job for applying the BDD format for the test, there are additional changes needed before forward the test. Would you please double checking and updating it? Thanks in advance.

**Pull request closed without merging**